src/Npgsql.NodaTime/Internal/NodaTimeTypeInfoResolver.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+using NodaTime;
+using Npgsql.Internal;
+using Npgsql.Internal.Converters;
+using Npgsql.PostgresTypes;
+using NpgsqlTypes;
+using static Npgsql.NodaTime.Internal.NodaTimeUtils;
+
+namespace Npgsql.NodaTime.Internal;
+
+sealed class NodaTimeTypeInfoResolver : IPgTypeInfoResolver
+{
+    static readonly DataTypeName TimestampTzDataTypeName = new("pg_catalog.timestamptz");
```

Comment:
```suggestion
    static readonly DataTypeName TimestampTzDataTypeName => new("pg_catalog.timestamptz");
```

and the others

Reply:
why? :)

Reply:
DataTypeNames also has these as props, less fields/mem, no static ctor. The string is a constant in the IL and promoted to the frozen heap on first use, the struct then is basically free too (though that doesn't matter here, as the resolver is not perf critical).

BCL/Runtime team did quite some work at some point to remove non-lazy static fields, to improve startup time.

Reply:
OK, that's quite a level of micro-optimization... Done.

Reply:
I agree but it's easy to keep consistent during a big rewrite ;)